operator_v2/validator/share_opt.go

```diff
@@ -21,56 +22,75 @@ type ShareOptions struct {
 	OperatorIds  []int          `yaml:"OperatorIds" env:"LOCAL_OPERATOR_IDS" env-description:"Local validator selected operator ids"`
```

Comment:
Operators & OperatorIds should be removed cause we can extract the data from the commitee

Reply:
remove not used ShareKey as well